docs/source/app-dev/services.rst

```diff
@@ -226,12 +226,13 @@ For full details, see :ref:`the proto documentation for the service <com.daml.le
 
 .. _ledger-identity-service:
 
-Ledger identity service
-=======================
+Ledger identity service (DEPRECATED)
+=====================================
 
 Use the **ledger identity service** to get the identity string of the ledger that your application is connected to.
 
-You need to include this identity string when submitting commands. Commands with an incorrect identity string are rejected.
+Including identity string is optional for all Ledger API requests.
+If you include it, commands with an incorrect identity string will be rejected.
```

Comment:
```suggestion
If you include it, commands with an incorrect identity string will be rejected. Note that even if you include it, the participant server may be configured to ignore it.
```

Reply:
This is not entirely true at the moment, but I believe it's the direction in which we aim to go, so maybe evaluate whether we want this in or not (/cc @meiersi-da).

Reply:
@stefanobaghino-da : I'm not sure. I would not add this text, as that is not what's implemented. We'd like our users to gradually stop using the field; and once we remove support for the field, then the participant server will effectively ignore its content.

Reply:
I understand, but that's why I stressed that it _may_ nbe configured to ignore it, stressing that ignoring it or not is up to the ledger implementation. I would bind us two ways to the current implementation. But again, your call, we can make a second change to the documentation once the implementation changes.

Reply:
Thanks. In that case, I'd opt to not add the text, as we don't explain to users how to figure out whether the server does do the check or not; and there may be some KV ledger users that do rely on the check happening.

Reply:
For the record, not applying this code suggestion